### PR TITLE
Hotfix for sphinx-init's generated requirements

### PR DESCRIPTION
**Pull request by felixfontein**
Fixes https://github.com/ansible-community/github-docs-build/issues/69.

CC @briantist

### CODE REVIEW

File: changelogs/fragments/2.0.0a2.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+release_summary: Hotfix to make ``sphinx-init`` work properly.
+bugfixes:
+  - "Bump version range of antsibull-docs requirement written by ``sphinx-init`` subcommand to ``>= 2.0.0a2, < 3.0.0`` (https://github.com/ansible-community/antsibull-docs/pull/149)."
```
Comment (gotmax23) [inline changelogs/fragments/2.0.0a2.yml:3]:
```suggestion
  - "Bump version range of antsibull-docs requirement written by ``sphinx-init`` subcommand to ``>= 2.0.0a2, < 3.0.0``. Previously, this was set to ``>=2.0.0, <3.0.0`` which could not be satisfied (https://github.com/ansible-community/antsibull-docs/pull/149)."
```

**Review by gotmax23 — APPROVED** [review]

**Review by gotmax23 — APPROVED** [review]
Ack. Thanks!

**Comment by gotmax23** [comment]
hmm, it looks like CI failed

**Comment by felixfontein** [comment]
Yeah, that's GitHub right now... there are random failures of GHA jobs to start up.

**Comment by felixfontein** [comment]
@gotmax23 thanks for reviewing this!

**Pull request merged**